src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
```

Comment:
```suggestion
    public abstract void doWork();
    
    public abstract void stopWork();
```
в класі / інтерфейсі методи при оголошенні розбиваються пустою строкою